Update android-APIReference.mdx (#12)
mirrorworld-universe/docs.mirrorworld.fun: data/android/android-APIReference.mdx

# Android API Reference
## Authentication Methods
### StartLogin

> Calling this api would popup a dialog, user can finish login flow on it. In which dialog, user can login with third method like google, twitter. Or he can login with his email which registered on our website.

Reviewer: ```suggestion
Calling this api would popup a dialog, user can finish login flow on it. In which dialog, user can login with third-party authentication like Google, Twitter. Or they can login with their email.
```
